src/MetaMorpho.sol

```diff
@@ -141,9 +141,17 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         _;
     }
 
-    /// @dev Reverts if the caller is not the `guardian`.
+    /// @dev Reverts if the caller is not the guardian.
     modifier onlyGuardian() {
-        if (_msgSender() != guardian) revert ErrorsLib.NotGuardian();
+        if (_msgSender() != owner() && _msgSender() != guardian) revert ErrorsLib.NotGuardian();
```

Comment:
IMHO, it would be much cleaner to separate the roles from the modifiers.

Otherwise, when I see a function with `onlyGuardian` I expect that only the `guardian` address can perform such action.

Reply:
I renamed this modifier to `onlyGuardianRole`, indicating that the role hierarchy also has power